licence-checker/licence-checker.py

```diff
@@ -60,15 +70,11 @@ def __init__(self, prefix, first_line_prefix):
         self.first_line_prefix = first_line_prefix
 
 
-class BlockCommentStyle(object):
+class BlockCommentStyle(LineCommentStyle):
```

Comment:
This doesn't look right! Looks like we need a `CommentStyle` class that's a supercass of both `LineCommentStyle` and `BlockCommentStyle`.